src/modules/check-box.js

```diff
@@ -0,0 +1,8 @@
+const checkBox = (box) => {
+  if (box.checked) {
+    return true;
+  }
+  return false;
+};
+
+export default checkBox;
```

Comment:
- [ ] When I make tasks as completed(By ticking the checkboxes) and refresh the page, I don't see the checkboxes ticked even I the local storage values are correct to true. Please make sure to fix this issue and refreshing the page should show tasks as completed(Ticked checkboxes) for tasks marked as completed before the page resfresh.

![image](https://user-images.githubusercontent.com/83803414/213467932-375a44ea-b307-4a76-8123-2b3b0af7e618.png)

Reply:
@thecodechaser But this is optional....is not on the requirement.

Reply:
Hello @RaphDasilva  this is part of the requirements. 

![image](https://user-images.githubusercontent.com/86318284/213522424-adcc3db1-0a46-4aff-b525-41c88fd3d2f1.png)

When a task is marked as completed, it is good to make sure that even if the user leaves now and comes back later and reloads, he/ she is supposed to still be able to see that a completed task is marked differently from other tasks.